TAGradingServer/account/admin-gradeable.php

```diff
@@ -0,0 +1,1124 @@
+HTML;
+        if($action==='edit' && !$g_grade_by_registration) {
+            print (isset($graders_to_sections[$fa_grader['user_id']])) ? $graders_to_sections[$fa_grader['user_id']] : '';
+        } 
+        else{
+            print $all_sections; 
+        }
+        print <<<HTML
+            "></td>
+        </tr>
+HTML;
+    }
+    
+    print <<<HTML
+            </table>
+            <br/>
+            <table>
+                <th>Limited Access Graders</th>
+HTML;
+
+   $db->query("SELECT user_id FROM users WHERE user_group=?", array(3));
+      
+    foreach($db->rows() as $la_grader){
+        print <<<HTML
+        <tr>
+            <td>{$la_grader['user_id']}</td>
+            <td><input style="width: 227px" type="text" name="grader-{$la_grader['user_id']}" class="graders" value="
+HTML;
+        if($action==='edit' && !$g_grade_by_registration) {
+            print (isset($graders_to_sections[$la_grader['user_id']])) ? $graders_to_sections[$la_grader['user_id']] : '';
+        } 
+        else{
+            print $all_sections; 
+        }
+        print <<<HTML
+"></td>
+        </tr>
+HTML;
+    }
+
+    print <<<HTML
+        </table>
+    </div> 
+        <br />
+HTML;
+
+    print <<<HTML
+            <!-- TODO default to the submission + late days for electronic -->
+            What date can the TAs start grading this?: <input name="date_grade" id="date_grade" class="datepicker" type="text"
+                style="cursor: auto; background-color: #FFF; width: 250px;">
+            
+            <br />
+            <!-- TODO default to never -->    
+            What date will the TA grades be released to the students? 
+            <input name="date_released" id="date_released" class="datepicker" type="text" 
+                   style="cursor: auto; background-color: #FFF; width: 250px;">    
+            
+            <br />
+            What syllabus/iris "bucket" does this item belong to?:
+            
+            <select name="gradeable-buckets" style="width: 170px;">
+                <!--<option value="homework"-->
+HTML;
+
+    $valid_assignment_type = array('homework','assignment','quiz','test','reading','participation',
+                                   'exam','lab','recitation','problem-set','project');
+    foreach ($valid_assignment_type as $type){
+        print <<<HTML
+                <option value="{$type}"
+HTML;
+        echo ($g_syllabus_bucket === $type)?'selected':'';
+        $title = ucwords($type);
+        print <<<HTML
+                >{$title}</option>
+HTML;
+    }
+    print <<<HTML
+            </select>
+            <br />
+            What is the lowest privileged user group that can grade this?
+            <select name="minimum-grading-group" style="width:180px;">
+HTML;
+
+    $grading_groups = array('1' => 'Instructor','2' => 'Full Access Grader','3' => 'Limited Access Grader', '4' => 'Student');
```

Comment:
I wouldn't include student right now as there's no way for a student to actually grade things.